lib/commands/fastboot.js

```diff
@@ -64,9 +64,8 @@ module.exports = {
     this.commandOptions = options;
 
     var runCommand = function() {
-      var appName = process.env.EMBER_CLI_FASTBOOT_APP_NAME || this.project.name();
-
-      return this.runCommand(appName, options);
+      var appOutputPaths = JSON.parse(process.env.EMBER_CLI_FASTBOOT_APP_OUTPUT_PATHS);
```

Comment:
This needs to have a fallback. Otherwise fastboot can only be used while building (since included hook is only ran for builds).

Maybe we should add a fastboot specific config file at config/fastboot.js for the shared config?